release/README.adoc

```diff
@@ -124,3 +124,67 @@ flowchart LR
 ----
 
 Note that the configuration of CircleCI *disallow* multiple build on the same branch (a new build cancel the previous one). It means that we have to run one step at a time.
+
+
+== 🏁 Run release alpha version pipeline
+
+Lunch only required steps in one CircleCI pipeline to release an alpha version. There are two methods available:
+
+* npm run
+[source, shell]
+----
+npm run am-release -- \
+  --branch master --gio-version 4.3.0 \
```

Comment:
mark somewhere that 4.3.0 needs to be changed to current release version

Reply:
it is only an example